### ISSUE TITLE
v2.0.0: multiple failures on Docker

### ISSUE THREAD

**Issue by ilike2burnthing** [FlareSolverr/FlareSolverr#202]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

### How to enable debug and html traces

[Follow the instructions from this wiki page](https://github.com/FlareSolverr/FlareSolverr/wiki/How-to-enable-debug-and-html-trace)

### Environment

* **FlareSolverr version**: v2.0.0
* **Last working FlareSolverr version**: v1.2.8
* **Operating system**: Docker (NAS)
* **Are you using Docker**: yes
* **FlareSolverr User-Agent (see log traces or / endpoint)**: Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0
* **Are you using a proxy or VPN?** no
* **Are you using Captcha Solver:** no
* **If using captcha solver, which one:** N/A
* **URL to test this issue:**
https://1337x.unblockit.kim/cat/Movies/time/desc/1/
https://badasstorrents.com/torrents/search/720p/date/desc
https://extratorrent.unblockit.kim/
https://idope.se/browse.html
https://limetorrents.unblockit.kim/latest100
https://torrentdownload.unblockit.kim/
https://torrentfunk.unblockit.kim/all/torrents/.html
https://yourbittorrent.nocensor.work/?v=&c=&q=2021

### Description

Different errors, often for the same URLs, with no successes. Jackett v0.18.1041.

Working fine when tested in Windows Sandbox.

### Logged Error Messages

```
2021-10-21T01:03:28+00:00 INFO REQ-1 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"},
2021-10-21T01:04:00+00:00 ERROR REQ-1 TimeoutError: Timed out after 30000 ms while trying to connect to the browser! Only Chrome at revision rlatest is guaranteed to work.
```
```
2021-10-21T01:18:19+00:00 INFO REQ-9 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"}
2021-10-21T01:20:00+00:00 ERROR REQ-9 Unexpected error: Error: Protocol error (Performance.enable): Target closed.,
2021-10-21T01:20:00+00:00 ERROR REQ-9 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
2021-10-21T01:20:00+00:00 INFO REQ-9 Response in 100.663 s
```
```
2021-10-21T01:35:30+00:00 INFO REQ-1 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"},
2021-10-21T01:35:50+00:00 INFO REQ-1 Cloudflare detected,
2021-10-21T01:36:14+00:00 INFO REQ-1 Challenge solved,
2021-10-21T01:36:18+00:00 INFO REQ-1 Response in 47.944 s

Jackett error:
The cookies provided by FlareSolverr are not valid: The cookies provided by FlareSolverr are not valid
```
```
2021-10-21T01:37:45+00:00 INFO REQ-2 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"},
2021-10-21T01:38:18+00:00 INFO REQ-2 Cloudflare detected,
2021-10-21T01:38:29+00:00 INFO REQ-2 Challenge solved,
2021-10-21T01:39:08+00:00 INFO REQ-2 Response in 83.119 s,
2021-10-21T01:39:08+00:00 ERROR REQ-2 Unexpected error: Error: Protocol error (Target.closeTarget): Target closed.,
2021-10-21T01:39:08+00:00 ERROR REQ-2 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
```
```
2021-10-21T01:46:22+00:00 INFO REQ-2 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"},
2021-10-21T01:47:13+00:00 INFO REQ-2 Cloudflare detected,
2021-10-21T01:47:18+00:00 INFO REQ-2 Challenge solved,
2021-10-21T01:47:44+00:00 INFO REQ-2 Response in 82.594 s,
2021-10-21T01:47:44+00:00 ERROR REQ-2 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms),
2021-10-21T01:47:44+00:00 ERROR REQ-2 Unexpected error: Error: Protocol error (Runtime.callFunctionOn): Target closed.
```
```
2021-10-21T01:26:36+00:00 INFO REQ-11 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://extratorrent.unblockit.kim/"},
2021-10-21T01:27:06+00:00 INFO REQ-11 Cloudflare detected,
2021-10-21T01:28:50+00:00 INFO REQ-11 Response in 133.555 s,
2021-10-21T01:28:50+00:00 ERROR REQ-11 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms),
2021-10-21T01:28:50+00:00 ERROR REQ-11 Unexpected error: Error: Protocol error (Network.getCookies): Session closed. Most likely the page has been closed.
```
```
2021-10-21T01:06:16+00:00 INFO REQ-4 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://idope.se/browse.html"}
2021-10-21T01:07:37+00:00 ERROR REQ-4 Unexpected error: Error: Protocol error (Page.setLifecycleEventsEnabled): Target closed.
2021-10-21T01:07:37+00:00 ERROR REQ-4 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
```
```
2021-10-21T01:08:00+00:00 INFO REQ-5 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://limetorrents.unblockit.kim/latest100"}
2021-10-21T01:09:24+00:00 INFO REQ-5 Response in 83.687 s
2021-10-21T01:09:24+00:00 ERROR REQ-5 Unexpected error: TimeoutError: Navigation timeout of 27500 ms exceeded
2021-10-21T01:09:24+00:00 ERROR REQ-5 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
```
```
2021-10-21T01:15:57+00:00 INFO REQ-8 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://yourbittorrent.nocensor.work/?v=&c=&q=2021"},
2021-10-21T01:17:17+00:00 ERROR REQ-8 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms),
2021-10-21T01:17:17+00:00 ERROR REQ-8 Unexpected error: Error: Navigation failed because browser has disconnected!,
2021-10-21T01:17:17+00:00 INFO REQ-8 Response in 79.969 s
```

Debug log:
```
,
> flaresolverr@2.0.0 start,
> node ./dist/server.js,
,
2021-10-21T01:51:27+00:00 INFO FlareSolverr v2.0.0,
2021-10-21T01:51:27+00:00 DEBUG Debug log enabled,
2021-10-21T01:51:27+00:00 INFO Testing web browser installation...,
2021-10-21T01:51:27+00:00 DEBUG Launching web browser...,
2021-10-21T01:52:51+00:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0,
2021-10-21T01:52:52+00:00 INFO Test successful,
2021-10-21T01:52:52+00:00 INFO Listening on http://0.0.0.0:8191,
2021-10-21T01:53:15+00:00 INFO REQ-1 Incoming request => GET /,
2021-10-21T01:53:15+00:00 INFO REQ-2 Incoming request => GET /favicon.ico,
2021-10-21T01:53:28+00:00 INFO REQ-3 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"},
2021-10-21T01:53:28+00:00 DEBUG REQ-3 Launching web browser...,
2021-10-21T01:54:01+00:00 DEBUG REQ-3 Navigating to... https://1337x.unblockit.kim/cat/Movies/time/desc/1/,
2021-10-21T01:54:12+00:00 DEBUG REQ-3 <!DOCTYPE html><html lang="en-US"><head>,
  <meta charset="UTF-8">,
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">,
  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">,
  <meta name="robots" content="noindex, nofollow">,
  <meta name="viewport" content="width=device-width,initial-scale=1">,
  <title>Just a moment...</title>,
  <style type="text/css">,
    html, body {width: 100%; height: 100%; margin: 0; padding: 0;},
    body {background-color: #ffffff; color: #000000; font-family:-apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, "Helvetica Neue",Arial, sans-serif; font-size: 16px; line-height: 1.7em;-webkit-font-smoothing: antialiased;},
    h1 { text-align: center; font-weight:700; margin: 16px 0; font-size: 32px; color:#000000; line-height: 1.25;},
    p {font-size: 20px; font-weight: 400; margin: 8px 0;},
    p, .attribution, {text-align: center;},
    #spinner {margin: 0 auto 30px auto; display: block;},
    .attribution {margin-top: 32px;},
    @keyframes fader     { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} },
    @-webkit-keyframes fader { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} },
    #cf-bubbles > .bubbles { animation: fader 1.6s infinite;},
    #cf-bubbles > .bubbles:nth-child(2) { animation-delay: .2s;},
    #cf-bubbles > .bubbles:nth-child(3) { animation-delay: .4s;},
    .bubbles { background-color: #f58220; width:20px; height: 20px; margin:2px; border-radius:100%; display:inline-block; },
    a { color: #2c7cb0; text-decoration: none; -moz-transition: color 0.15s ease; -o-transition: color 0.15s ease; -webkit-transition: color 0.15s ease; transition: color 0.15s ease; },
    a:hover{color: #f4a15d},
    .attribution{font-size: 16px; line-height: 1.5;},
    .ray_id{display: block; margin-top: 8px;},
    #cf-wrapper #challenge-form { padding-top:25px; padding-bottom:25px; },
    #cf-hcaptcha-container { text-align:center;},
    #cf-hcaptcha-container iframe { display: inline-block;},
  </style>,
,
      <meta http-equiv="refresh" content="35">,
  <script type="text/javascript">,
    //<![CDATA[,
    (function(){,
      ,
      window._cf_chl_opt={,
        cvId: "2",,
        cType: "non-interactive",,
        cNounce: "57562",,
        cRay: "6a16db0d3a9c9c57",,
        cHash: "da9ecd8e64030a6",,
        cFPWv: "b",,
        cTTimeMs: "1000",,
        cRq: {,
          ru: "aHR0cHM6Ly8xMzM3eC51bmJsb2NraXQua2ltL2NhdC9Nb3ZpZXMvdGltZS9kZXNjLzEv",,
          ra: "TW96aWxsYS81LjAgKFgxMTsgTGludXggeDg2XzY0OyBydjo3OC4wKSBHZWNrby8yMDEwMDEwMSBGaXJlZm94Lzc4LjA=",,
          rm: "R0VU",,
          d: "+HNB5nhzU+Ooui3YNNGLAOpBHq7m/+kLxGbkMbSBYLbGSgqUuUhGBE2wxmx/5nrhytpDQjtfIxi82xkdeLTlTafWXGxdTjgfHPL2sZ0CWYkNltHqK1m4WeWIJZ+fzNt2XOnIXW/fUnY6dCYbOExC4eUEZb4GwOOvyl0CVeEe2rVd4lSoI6o1s2Duz4Hm5Rh4W/bzYsWCZv/J8EDn2bhctkT+myyTmpsK2EvKuVOUcoYrpa9FrsIMGafaXM9flGdMhyXH+Hslw8lbiDnoFKfmIRCBb0rmoDHQuFkrRTDdBgvPWhkIEiUGRRofKXWQAjIiyz198AEcsind9JujHMuzF4oFOcyRjm7HBekUU22wyh2bQNopV2ltY0vRDRR08LQcebTB9NDYDA1dH4Jt0zfdhrmuYCfcL/3w/k1rYy+RN7eRBJMXNzFo8/0TGGO2LO3OY07Vkjh+1Yb+HxSBRRDFHCPRVaP5xPljhuHwaqqrt+A=",,
          t: "MTYzNDc4MTI0Mi40NDkwMDA=",,
          m: "VCXtdHFvfOsDgh12PjDFn1wpdPOEYk/nb7RtVDCYF8Y=",,
          i1: "cqbwf0y8w53iHWVkUAGqiA==",,
          i2: "Js4VMvKtPrrl3bZ0UFXddQ==",,
          zh: "F+k/vfjUgJKigNcYHWLh+AQFKMOuvaR5DqUjVmBswYk=",,
          uh: "kLjFiOWZAmwM1tw9xWESQZ48+G3DKCBRmJlMtnmJe0E=",,
          hh: "JyehVYAjtmTxWSEl6DQEF3rBmrBz4RpngvqvBCwIc8A=",,,
},
      },
      window._cf_chl_enter = function(){window._cf_chl_opt.p=1};,
      ,,
;,,
>,
  </script>,
  ,
,
<script type="text/javascript" src="/cdn-cgi/challenge-platform/h/b/orchestrate/jsch/v1?ray=6a16db0d3a9c9c57"></script></head>,
<body>,
  <table width="100%" height="100%" cellpadding="20">,
    <tbody><tr>,
      <td valign="middle" align="center">,
          <div class="cf-browser-verification cf-im-under-attack">,
  <noscript>,
    <h1 data-translate="turn_on_js" style="color:#bd2426;">Please turn JavaScript on and reload the page.</h1>,
  </noscript>,
  <div id="cf-content" style="display: block;">,
    ,
    <div id="cf-bubbles">,
      <div class="bubbles"></div>,
      <div class="bubbles"></div>,
      <div class="bubbles"></div>,
    </div>,
    <h1><span data-translate="checking_browser">Checking your browser before accessing</span> unblockit.kim.</h1>,
    ,
    <div id="no-cookie-warning" class="cookie-warning" data-translate="turn_on_cookies" style="display:none">,
      <p data-translate="turn_on_cookies" style="color:#bd2426;">Please enable Cookies and reload the page.</p>,
    </div>,
    <p data-translate="process_is_automatic">This process is automatic. Your browser will redirect to your requested content shortly.</p>,
    <p data-translate="allow_5_secs" id="cf-spinner-allow-5-secs">Please allow up to 5 seconds…</p>,
    <p data-translate="redirecting" id="cf-spinner-redirecting" style="display:none">Redirecting…</p>,
  </div>,
   ,
  <form class="challenge-form" id="challenge-form" action="/cat/Movies/time/desc/1/?__cf_chl_jschl_tk__=pmd_St0X_EOyyUl5snsqfKtUvYWX7z2KjAnG6HMugPEUCSU-1634781242-0-gqNtZGzNAhCjcnBszQe9" method="POST" enctype="application/x-www-form-urlencoded">,
    <input type="hidden" name="md" value="ljXIjIBcfcHmGILvXc2ZiRTiKDpY9fy8.HxaNIG2Kpc-1634781242-0-AcBfCzV7stdMTOxJZ2-Tm4HxJBo9Gr458oSuGbxXvCm4bI2FCaptOFPhLHrrA0wgRri2b_pAzioUaEQWhkgjwddF0zBoecqwpiowHYANOsu_EdzU70pBIyAFHwDzU34ulD23wdIJ3YhUMKR9dSosswFc7zzYeHm4LVX3cJb__OkxoQGP2edHHjF9eMuyzFpxvTEahoEeMFHbuWs5Cwoa_wqBqcpF60bjUO1dpSRiDaSkyKTozBU6VMfBfOXaZguMz5cxypdvkUFDpFOAuQVy7F2wy5YbYvbB-Q9GI5-a3Qu9LX1Cjyi9-s8AVW5dkuWRiwF3TYyB4I8keMJBlmqulWPyhhP-62C2h4J8VgrzhmxjWEVyEhhs5ratqjSYxDzQIQpx7fZj6iKjHLvsgx0LZzl9iXfOOYeCTt4JjmRZVvq7tFVZebtKJuffB8qhIJ3HOCdcL-b27lHsCwgFxDL1pZo">,
    <input type="hidden" name="r" value="ioIsQFYmc6UYheg92qq4s.hd75wNqfUCvL4N6vUpsWc-1634781242-0-AVQvOnbJ7+RIZWtVmKSk8glclWwakFwHyzC7fNwPfUSZLY+ds8W1zecjFdzk9ucAiuSfZVQP2orh5wh0TAsRIbKLns+eu63ww94hlb4F9tNY18DJKWKncJ/Kq8yNMmAYx2ffeQ+tL+sp0pvLKSozkUH3PQ112p+OGR4VuKMHR9FaFsN5zPdeM2PokVw+Yx10P36gPbw2alUkt2BNdEvnLvVf63oQeyj5NL3EvpxR52dWNQNGNgKJmb0l7ehcXFHlcsUKqj65ax22UrLzeZ46nd3mGn7TgY9ei3A1QxnKN2JTGSzO1k/54OV1D96SZMyo0m+64ZTBAIWrXjGlxokAhgU75j9f9Ri70pDLVJ8lDuyMLYjfsVk3BX7OyhjNdUTQQHQqMHC27M79gP0llifLXduy7sov2k2neGvJQIAggxFQMAQR5PcCLalKIAE7U/8/NyAxTFZ1uc0ke6ka1lKkzU7CTFeL4MFmtTkwhGU2X8NGVSfGUNp7LWM8bE+hna5u+MsxXph9eNTd/+QwyOkFwk1/y+qqvWjwZVfcQ5yl8iT7ojY5DxI0g8ppnqjxaBZaWqlJAxrHWGf+L55OFans5oZ0+0bVU9cvWg7jlPzaV+pVE4zYlMdUCDucBz+DQzKOBt4P9q6VYvgdnMynkUsnMLRYWqI29u1kMKQKppi6A0S0pg6rejfF9iGxdfME+LpMjhKepLu0LD+MAul/n3WQecfUkudRKhGsBHW7zNTWt9xnvO9B3+Vk63bDjTDVDDFRrUYnrN5dswqjkGfK91XeErgnJYSfZFNhFe9Hroi5yqQ9y5mjGJ/vBdm8p22wYTkj95I/4qbswyIualk6XgQlHuh8QvzKSa0JfMLzcWPkcINiqfi5Bd74rZzLBWDCaA/Zs3q5a/PItbsRh999LlZs9HmOhLfIAgpoxhzHIerblHacTXNJHsCht6FlQvjDB5avV5f53fqISiuvz0aTbvlAw0N30rArxAsG8464LjjHWIfu6+3Q/gyoooHj5tA8kTakfdKGIYusqaMMThkbN3/vIyjxAD95ZMaB3piZCwUfuhZEKZLzdEznISwwB+5l4/zSko7LewyqqQ79lQiFKb47QrCKkHxAyq1D0Y7ydfp8xm43Eell2uK61gBvLwHy5eLijOivmgbNqfHD3bzBAJ1syUw4iDVRNcj5VsDxoDis4S+nGQua4JJEKtkLW4wvL8uJyIr9QqhLSGHTyy4xFmx7PB363gqk3KFsA60b0Ou33xae/G/Iz5LYJXYtrbsgM3Rz9pG2KWsmFen0gOf+6qlKx5gCyXbkyjhNEg0mwLoxatv0vgXLLccHqiE7LO7VqTlKdt4BELvuKaq2/UlZK0teKCaFTVu2kx9e1lVggtf7UG9fpnTaY6RNBywrpdpxnFoYaNb5LKqoK/RAfCE2BjKj1i4hm9VjqNKf0ZDmSLw68762373MhbHVAEDmlz+buNfWktqonEJv7Aga7ZRA4nMbOpSVoHugwj0J7zKT5UvQUFLbUVm1I/sLxLRA7Fmx8/fl+TrYp8xe2BcURkaq2QVBwhS1DQFXk/xKlVLqiImv1uDVROggdKMH4n9c8KDSbyn8TrMeZXTebBoWqwmVX5CYD0CT3gPT8YPHf53G2/k6kP2ilg+7SYzK0lrLxbQtSAqekcP3oboRZBNSPjlKmQ6sq2z33lPi6zwtHmp/F4zzZV0DUBRZ1hoHVB7kD85kWvEg1GgVh5/uL2uvGnxGWygOtaSlBxPlA9sg3mkNlBXTxdwAfm8PymMvodlsothVgtO3hv2qDvTjq1DJAAgywWtStFUPHReK5YRM11KXvN9MGssY5LJGEE0jhIlaJKLEgIzRIc0xL7mDfW3xdL+sZnv3Tnb+5oxSmkaFnklspLgHqOqBJ53dWDKwky6fPWt6ejuX7A==">,
    <input type="hidden" value="d50f5399e5bdfc737dbbe5ba84ce5440" id="jschl-vc" name="jschl_vc">,
    <!-- <input type="hidden" value="" id="jschl-vc" name="jschl_vc"/> -->,
    <input type="hidden" name="pass" value="1634781243.449-m4ApypCsE3">,
    <input type="hidden" id="jschl-answer" name="jschl_answer">,,
>,
     ,
    <script type="text/javascript">,
      //<![CDATA[,
      (function(){,
          var a = document.getElementById('cf-content');,
          a.style.display = 'block';,
          var isIE = /(MSIE|Trident\/|Edge\/)/i.test(window.navigator.userAgent);,
          var trkjs = isIE ? new Image() : document.createElement('img');,
          trkjs.setAttribute("src", "/cdn-cgi/images/trace/jschal/js/transparent.gif?ray=6a16db0d3a9c9c57");,
          trkjs.id = "trk_jschal_js";,
          trkjs.setAttribute("alt", "");,
          document.body.appendChild(trkjs);,
          var cpo=document.createElement('script');,
          cpo.type='text/javascript';,
          cpo.src="/cdn-cgi/challenge-platform/h/b/orchestrate/jsch/v1?ray=6a16db0d3a9c9c57";,
          document.getElementsByTagName('head')[0].appendChild(cpo);,
        }());,
      //]]>,
    </script>,
  ,
,
  ,
  <div id="trk_jschal_nojs" style="background-image:url('/cdn-cgi/images/trace/jschal/nojs/transparent.gif?ray=6a16db0d3a9c9c57')"> </div>,
</div>,
,
          ,
          <div class="attribution">,
            DDoS protection by <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing/" target="_blank">Cloudflare</a>,
            <br>,
            <span class="ray_id">Ray ID: <code>6a16db0d3a9c9c57</code></span>,
          </div>,
      </td>,
     ,,
>,
  </tbody></table><img src="/cdn-cgi/images/trace/jschal/js/transparent.gif?ray=6a16db0d3a9c9c57" id="trk_jschal_js" alt="">,
,
,
</body></html>,
2021-10-21T01:54:13+00:00 INFO REQ-3 Cloudflare detected,
2021-10-21T01:54:15+00:00 DEBUG REQ-3 Javascript challenge element '#trk_jschal_js' detected.,
2021-10-21T01:54:15+00:00 DEBUG REQ-3 Waiting for Cloudflare challenge...,
2021-10-21T01:54:20+00:00 DEBUG REQ-3 Javascript challenge element '#trk_jschal_js' detected.,
2021-10-21T01:54:21+00:00 DEBUG REQ-3 Challenge element is visible,
2021-10-21T01:54:21+00:00 DEBUG REQ-3 Found challenge element again,
2021-10-21T01:54:21+00:00 DEBUG REQ-3 Waiting for Cloudflare challenge...,
2021-10-21T01:54:29+00:00 DEBUG REQ-3 Javascript challenge element '#trk_jschal_js' detected.,
2021-10-21T01:54:29+00:00 DEBUG REQ-3 Challenge element is visible,
2021-10-21T01:54:29+00:00 DEBUG REQ-3 Found challenge element again,
2021-10-21T01:54:29+00:00 DEBUG REQ-3 Waiting for Cloudflare challenge...,
2021-10-21T01:54:31+00:00 DEBUG REQ-3 Javascript challenge element '#trk_jschal_js' detected.,
2021-10-21T01:54:31+00:00 DEBUG REQ-3 Challenge element is visible,
2021-10-21T01:54:31+00:00 DEBUG REQ-3 Found challenge element again,
2021-10-21T01:54:31+00:00 DEBUG REQ-3 Waiting for Cloudflare challenge...,
2021-10-21T01:54:33+00:00 DEBUG REQ-3 Javascript challenge element '#trk_jschal_js' detected.,
2021-10-21T01:54:34+00:00 DEBUG REQ-3 Challenge element is visible,
2021-10-21T01:54:34+00:00 DEBUG REQ-3 Found challenge element again,
2021-10-21T01:54:34+00:00 DEBUG REQ-3 Waiting for Cloudflare challenge...,
2021-10-21T01:54:44+00:00 ERROR REQ-3 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms),
2021-10-21T01:54:44+00:00 INFO REQ-3 Response in 75.62 s,
2021-10-21T01:54:44+00:00 DEBUG REQ-3 Unexpected error: Error: Protocol error (Runtime.callFunctionOn): Target closed.,
2021-10-21T01:54:44+00:00 DEBUG REQ-3 Validating HTML code...,
2021-10-21T01:54:44+00:00 ERROR REQ-3 Unexpected error: Error: Protocol error (Network.getCookies): Session closed. Most likely the page has been closed.
```

### Screenshots

N/A

**Comment by ilike2burnthing** [comment]
I redeployed the container and got similar errors, but it eventually resolved. Still errors out when using Jackett though.

```
> flaresolverr@2.0.0 start,
> node ./dist/server.js,
2021-10-21T04:23:23+00:00 INFO FlareSolverr v2.0.0,
2021-10-21T04:23:23+00:00 INFO Testing web browser installation...,
/home/node/flaresolverr/node_modules/puppeteer/lib/launcher/BrowserRunner.js:170,
            reject(new Errors_1.TimeoutError(`Timed out after ${timeout} ms while trying to connect to the browser! Only Chrome at revision r${preferredRevision} is guaranteed to work.`));,
                   ^,
TimeoutError: Timed out after 30000 ms while trying to connect to the browser! Only Chrome at revision rlatest is guaranteed to work.,
    at Timeout.onTimeout (/home/node/flaresolverr/node_modules/puppeteer/lib/launcher/BrowserRunner.js:170:20),
    at listOnTimeout (node:internal/timers:557:17),
    at processTimers (node:internal/timers:500:7),
npm notice ,
npm notice New minor version of npm available! 8.0.0 -> 8.1.0,
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.1.0>,
npm notice Run `npm install -g npm@8.1.0` to update!,
npm notice ,
> flaresolverr@2.0.0 start,
> node ./dist/server.js,
2021-10-21T04:24:14+00:00 INFO FlareSolverr v2.0.0,
2021-10-21T04:24:14+00:00 INFO Testing web browser installation...,
node:internal/process/promises:246,
          triggerUncaughtException(err, true /* fromPromise */);,
          ^,
Error: NS_ERROR_ABORT at https://www.google.com,
    at navigate (/home/node/flaresolverr/node_modules/puppeteer/lib/FrameManager.js:95:23),
    at processTicksAndRejections (node:internal/process/task_queues:96:5),
    at async FrameManager.navigateFrame (/home/node/flaresolverr/node_modules/puppeteer/lib/FrameManager.js:70:21),
    at async Frame.goto (/home/node/flaresolverr/node_modules/puppeteer/lib/FrameManager.js:295:16),
    at async Page.goto (/home/node/flaresolverr/node_modules/puppeteer/lib/Page.js:485:16),
    at async testWebBrowserInstallation (/home/node/flaresolverr/dist/services/sessions.js:49:5),
  -- ASYNC --,
    at Frame.<anonymous> (/home/node/flaresolverr/node_modules/puppeteer/lib/helper.js:94:19),
    at Page.goto (/home/node/flaresolverr/node_modules/puppeteer/lib/Page.js:485:53),
    at Page.<anonymous> (/home/node/flaresolverr/node_modules/puppeteer/lib/helper.js:95:27),
    at testWebBrowserInstallation (/home/node/flaresolverr/dist/services/sessions.js:49:16),
    at processTicksAndRejections (node:internal/process/task_queues:96:5),
> flaresolverr@2.0.0 start,
> node ./dist/server.js,
2021-10-21T04:25:49+00:00 INFO FlareSolverr v2.0.0,
2021-10-21T04:25:49+00:00 INFO Testing web browser installation...,
/home/node/flaresolverr/node_modules/puppeteer/lib/launcher/BrowserRunner.js:170,
            reject(new Errors_1.TimeoutError(`Timed out after ${timeout} ms while trying to connect to the browser! Only Chrome at revision r${preferredRevision} is guaranteed to work.`));,
                   ^,
TimeoutError: Timed out after 30000 ms while trying to connect to the browser! Only Chrome at revision rlatest is guaranteed to work.,
    at Timeout.onTimeout (/home/node/flaresolverr/node_modules/puppeteer/lib/launcher/BrowserRunner.js:170:20),
    at listOnTimeout (node:internal/timers:557:17),
    at processTimers (node:internal/timers:500:7),
> flaresolverr@2.0.0 start,
> node ./dist/server.js,
2021-10-21T04:26:35+00:00 INFO FlareSolverr v2.0.0,
2021-10-21T04:26:35+00:00 INFO Testing web browser installation...,
2021-10-21T04:27:46+00:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0,
2021-10-21T04:27:49+00:00 INFO Test successful,
2021-10-21T04:27:49+00:00 INFO Listening on http://0.0.0.0:8191
```

**Comment by ocalvo** [comment]
I rather wait for a fix in the docker image.
This is a Windows Server Core installation and I don't install any software in the actual Windows Server baremetal, it needs to stay as clean as possible.

**Comment by ocalvo** [comment]
> @ocalvo I don't have a Windows system to tests Linux containers for Windows. I guess it can be related to the user used inside the Docker container but there are a lot of things that could be. I know the containers are working well in Linux because there are few issues with 100.000 downloads of the Docker image yesterday.
> 
> You can try native binary for Windows => https://github.com/FlareSolverr/FlareSolverr/releases/download/v2.0.0/flaresolverr-v2.0.0-windows-x64.zip

I have reverted back to ghcr.io/flaresolverr/flaresolverr:v1.2.9. 
This version does work fine for me.

**Comment by ilike2burnthing** [comment]
**Relative idle**
System CPU - ~30%
System Memory - ~60%
Container CPU - 0%
Container Memory - 22.4MB, 16.5MB cache

**FlareSolverr processing Jackett request**
System CPU - 40-60%
System Memory - 60-70%
Container CPU - spikes ranged from 60% to 120%
Container Memory - 185-255MB, 125-190MB cache

**Error**
```
2021-10-22T03:51:18+00:00 INFO REQ-3 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"}
2021-10-22T03:52:03+00:00 INFO REQ-3 Cloudflare detected
2021-10-22T03:52:29+00:00 INFO REQ-3 Challenge solved
2021-10-22T03:52:37+00:00 ERROR REQ-3 Unexpected error: Error: Protocol error (Network.getCookies): Session closed. Most likely the page has been closed.
2021-10-22T03:52:37+00:00 ERROR REQ-3 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
2021-10-22T03:52:37+00:00 INFO REQ-3 Response in 79.394 s
```

Also came across a new error at one point:
```
2021-10-22T03:41:41+00:00 INFO REQ-2 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://1337x.unblockit.kim/cat/Movies/time/desc/1/"}
2021-10-22T03:42:31+00:00 INFO REQ-2 Cloudflare detected
2021-10-22T03:42:50+00:00 INFO REQ-2 Cloudflare Error: Protocol error (Runtime.callFunctionOn): Cannot find context with specified id callFunctionOn@chrome://remote/content/domains/content/Runtime.jsm:199:15
execute@chrome://remote/content/domains/DomainCache.jsm:99:25
receiveMessage@chrome://remote/content/sessions/ContentProcessSession.jsm:86:45
2021-10-22T03:42:50+00:00 INFO REQ-2 Response in 68.637 s
```

Container's memory reservation, memory limit, and CPU limit are all set to `unlimited`.

**Comment by ngosang** [comment]
I released v2.0.2 with a clarification message about CPU limits.
